src/js/post-911-gib-status/components/UserInfoSection.jsx

```diff
@@ -14,17 +14,20 @@ class UserInfoSection extends React.Component {
     const percentageBenefit = formatPercent(enrollmentData.percentageBenefit) || 'unavailable';
     const fullName = `${enrollmentData.firstName} ${enrollmentData.lastName}`;
 
-    // TODO: Figure out what will be sent to us if the user is ineligible so we can
-    // conditionally show "Currently Disallowed" message
-
-    return (
-      <div>
-        <h3 className="section-header">Chapter 33 Benefit Information</h3>
+    let currentOfAlert;
```

Comment:
Can we name these `currentAsOfAlert` and `showCurrentAsOfAlert`? Still a little awkward but I think more self-describing.